### DISCUSSION TITLE
Allow hardpoint customization in multiplayer lobby for mission & other gamemodes

### DISCUSSION THREAD

**Discussion by ahzidaljun** [FakeFishGames/Barotrauma#6274] [Ideas and suggestions]
If possible, this would be nice for testing custom subs and allowing the use of weaker campaign-oriented subs in more demanding difficulties. This could be extended to other gamemodes as applicable, like PvP and sandbox.

![Screenshot_50](https://user-images.githubusercontent.com/72061961/124808789-c10c4680-df35-11eb-8303-1e562a382c53.png)

(sorry for posting this in the wrong section initially haha)

**Comment by Ek-Videogames** [comment]
+1
Especially good if people wanted running mission-servers with subs like Dugong, Orca, Big-Rig, that require hardpoints being filled.
